.github/labeler.yml

```diff
@@ -0,0 +1,29 @@
+documentation:
+- any:
+  - changed-files:
+    - all-globs-to-all-files: ['docs/**', '!lib/rules/**']
+
+rule:
+- any:
+  - changed-files:
+    - any-glob-to-any-file: ['lib/rules/**']
+
+cli:
+- any:
+  - changed-files:
+    - any-glob-to-any-file: ['lib/cli.js', 'lib/options.js', 'lib/cli-engine/**']
```

Comment:
We probably also want `lib/eslint/**` to capture `ESLint` changes.

Reply:
Added 👍🏻

Reply:
Sorry, I meant that `lib/eslint/**` should be under `cli` because that really is the driver of the CLI.

Reply:
Oh! Updated 👍🏻